CONTRIBUTING.md

```diff
@@ -9,19 +9,35 @@ To start the local site use:
 To deploy updated versions:
 `npm run deploy`
 
-To run all tests via Jest:
+To run all tests via [Jest](https://jestjs.io/):
 `npm test`
 
-To run Prettier via command line:
+To run [Prettier](https://prettier.io/) via command line:
 `npm run format`
 
 To check if your files are formatted:
 `npm run format:check`
 
+To run the linter:
+`npm run lint`
+
 # Using Cloudflare Workers
 
 1. Run `npx wrangler dev <filepath>` in your terminal to start a development server
 1. Open a browser tab at http://localhost:8080/ to see your worker in action
 1. Run `npx wrangler publish <file-path> --name <worker-name>` to publish your worker
 
 Learn more at https://developers.cloudflare.com/workers/
+
```

Comment:
Is it possible to restructure the document so that the "How to Contribute" section appears before "Getting Started"?